src/mixer/basetrackplayer.cpp

```diff
@@ -44,6 +45,15 @@ BaseTrackPlayerImpl::BaseTrackPlayerImpl(QObject* pParent,
     m_pChannel = new EngineDeck(channelGroup, pConfig, pMixingEngine,
                                 pEffectsManager, defaultOrientation);
 
+    m_pInputEnabled.reset(new ControlObjectSlave(group, "input_enabled", this));
+    m_pPassthroughEnabled.reset(new ControlObjectSlave(group, "passthrough", this));
+    m_pPassthroughEnabled->connectValueChanged(SLOT(slotPassthroughEnabled(double)));
```

Comment:
Does this connection always work here? 
It looks like we are starting to clutter our CO interface with  "input_enabled" "passthrough" and "enabled". 
What should be used when? IMHO we should use passthough only for the vinyl control case. It this still the case?